metrics-sender/src/main/java/io/knotx/metrics/graphite/GraphiteReporter.java

```diff
@@ -0,0 +1,255 @@
+package io.knotx.metrics.graphite;
```

Comment:
If part of this code is copied from somewhere, I believe there should be a licence header, that says on what license the original code was distributed. And there should be license header for this project.

If would be very convenient (if possible) to list changes done in comparison to the original source in the comment.

Reply:
graphite-metrics uses the same license Apache 2.0, so I added a common header.
The changes information has been added.